Packages/CommonUI/CommonUI/Classes/View/TextField/SecureTextField.swift

```diff
@@ -134,6 +144,8 @@ private extension SecureTextField {
 
 struct SecureTextField_Previews: PreviewProvider {
 	static var previews: some View {
-		SecureTextField(secureText: .constant("Test"), inputStyle: .constant(.error(message: "Error")), placeHolder: "Phone")
+		SecureTextField(secureText: .constant("Test"), inputStyle: .constant(.error(message: "Error")), placeHolder: "Phone"){ _ in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/f6cf3ec46c326a63497b0525c2c376cdbb03e8f1/Packages/CommonUI/CommonUI/Classes/View/TextField/SecureTextField.swift#L147"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>SecureTextField.swift:147</code></td>
    </tr>
  </tbody>
</table>

Reply:
fix complete